app_dart/lib/src/service/config.dart

```diff
@@ -317,6 +317,7 @@ class Config {
         'skia-flutter-autoroll',
         'engine-flutter-autoroll',
         'dependabot',
+        'dependabot[bot]',
```

Comment:
Do we need to remove the one above?

Reply:
We do have earlier PRs based on `dependabot` which didn't complain about tests: https://github.com/flutter/engine/pull/40447